fineract-provider/src/main/java/org/apache/fineract/portfolio/calendar/domain/Calendar.java

```diff
@@ -387,7 +395,13 @@ public static Calendar fromJson(final JsonCommand command) {
             actualChanges.put(secondRemindarParamName, newValue);
             this.secondReminder = newValue;
         }
-
+        
+        if (command.isChangeInStringParameterNamed(CALENDAR_SUPPORTED_PARAMETERS.MEETING_TIME.getValue(), this.meetingtime.toString())) {
+            final String newValue = command.stringValueOfParameterNamed(CALENDAR_SUPPORTED_PARAMETERS.MEETING_TIME.getValue());
+            actualChanges.put(CALENDAR_SUPPORTED_PARAMETERS.MEETING_TIME.getValue(), newValue);
+            this.meetingtime =Time.valueOf(StringUtils.defaultIfEmpty(newValue, null));
```

Comment:
Don't use java.sql.Time to parse time data. Please add a method in JsonParserHelper to get the time object. Please have a look how LocalDate is parsed in the same file.